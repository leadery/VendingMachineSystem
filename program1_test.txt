LOGIN alice927 654321
LOGOUT
LOGIN alicelee0927 123456
LOGOUT
LOGIN alice927 zheshiyigetebiechangdemima
LOGOUT
LOGIN frank420 sleepall
BUY A1
LOGOUT
LOGIN alice927 123456
DEPOSIT 400
DEPOSIT $400
BALANCE
BUY A3D
PRICE A20
BUY A1
PRICE A2
EXIT
