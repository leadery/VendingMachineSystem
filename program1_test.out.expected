USER LOGIN MODE
PLEASE ENTER YOUR REQUEST
USER LOGIN MODE
ERROR INVALID INPUT
ERROR INVALID INPUT
ERROR INVALID INPUT
ERROR INVALID INPUT
PLEASE ENTER YOUR REQUEST
USER LOGIN MODE
PLEASE ENTER YOUR REQUEST
ERROR INVALID INPUT
$400 DEPOSITED ~~~ BALANCE NOW $1400
INSUFFICIENT BALANCE
SODA IS $75
BALANCE NOW $1400
ERROR INVALID INPUT
REQUESTED ITEM IS NOT AVAIBLE
ITEM SODA BOUGHT FOR $75
